frontend/src/components/ExternalDb.vue

```diff
@@ -27,14 +27,24 @@
         </li>
       </ul>
     </div>
+    <div v-else>
+      <p class="notification has-background-danger-light" style="margin-top: 5%;" v-html="errorMessage"></p>
```

Comment:
```suggestion
      <p class="notification has-background-danger-light mt-6" v-html="errorMessage"></p>
```

Not sure exactly what we decided on avoiding inline styles, but is this an okay replacement?

Reply:
Good point. I've changed it in a2b055cc